Tests/js/views/services/plugins/assets/ez-locationsearchplugin-tests.js

```diff
@@ -335,7 +388,20 @@ YUI.add('ez-locationsearchplugin-tests', function (Y) {
         },
 
         "Should handle loading content error": function () {
-            var resultAttr = 'whateverAttr';
+            var resultAttr = 'whateverAttr',
+            that = this;
+
+            Mock.expect(this.contentService, {
+                method: 'createView',
+                args: [Mock.Value.Object, Mock.Value.Function],
+                run: function (query, cb) {
+                    if (query == that.locationQuery) {
+                        cb(false, that.locationResponse);
+                    } else if (query == that.contentQuery) {
+                        cb(true, that.contentResponse);
+                    }
+                }
+            });
 
             Y.eZ.Content = function () {};
```

Comment:
same as https://github.com/ezsystems/PlatformUIBundle/pull/492/files#r49862676 for those 2 lines.